metadata/Cadastre/basic_parcels/kane_county_parcels_schema.md

```diff
@@ -0,0 +1,15 @@
+# Title
+
+Kane County Parcels Field Definitions
+
+## ID
+
+0836837f-cbfc-4ae5-84f3-25befd6f9937
+
+### ZONING
+
+<!--- No definition for this field --->
+
+### ACCOUNT_NUM
```

Comment:
I think we can remove ACCOUNT_NUM now that it has been added to all the counties and has become part of the Basic parcel schema. The Zoning field however is unique to Kane